app-config-electron/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@app-config/electron",
```

Comment:
There are a few places we should add this package, `new-version.sh`, `publishing.yml`, `CONTRIBUTING.md`, maybe one I'm not thinking about?